src/controllers/Search-Controller.coffee

```diff
@@ -78,11 +81,13 @@ class SearchController
                 if (@searchData.results?)
                   @res.send(@renderPage())
                 else
+                  logger?.info {Error:'There are no results that match your search.'}
```

Comment:
It is internal. How  about?

logger?.info {Error:'There are no results that match the search.',queryId: queryId, filters:filters}